internal/adapters/clients/ollama.go

```diff
@@ -11,110 +11,23 @@ import (
 	"github.com/sashabaranov/go-openai"
 )
 
+// NewOllamaClient constructs a new OllamaClient with the provided configuration.
 func NewOllamaClient(cfg openai.ClientConfig) *OllamaClient {
 	return &OllamaClient{
 		cfg: cfg,
 	}
 }
 
+// OllamaClient manages chat completions for the Ollama model.
 type OllamaClient struct {
 	cfg openai.ClientConfig
```

Comment:
For the 'OllamaClient' struct, it's generally good practice to also document the fields within the struct, especially if they're exported or crucial to the understanding of its operations.